dist/images/ovnkube.sh

```diff
@@ -68,11 +68,11 @@ fi
 cmd=${1:-""}
 
 # ovn daemon log levels
-ovn_log_northd=${OVN_LOG_NORTHD:-"-vconsole:info"}
-ovn_log_nb=${OVN_LOG_NB:-"-vconsole:info"}
-ovn_log_sb=${OVN_LOG_SB:-"-vconsole:info"}
-ovn_log_controller=${OVN_LOG_CONTROLLER:-"-vconsole:info"}
-ovn_log_nbctld=${OVN_LOG_NBCTLD:-"-vfile:info"}
+ovn_loglevel_northd=${OVN_LOGLEVEL_NORTHD:-"-vconsole:info"}
```

Comment:
i think we should change the default values to -vfile:info, because we start all the processes with --logfile, so -vconsole:info is pretty much useless here. @girishmg thoughts?